### YamlMime:ManagedReference
items:
- uid: System.Windows.Forms.DataGridViewDataErrorEventArgs
  id: DataGridViewDataErrorEventArgs
  children:
  - System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)
  - System.Windows.Forms.DataGridViewDataErrorEventArgs.Context
  - System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception
  - System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException
  langs:
  - csharp
  name: DataGridViewDataErrorEventArgs
  nameWithType: DataGridViewDataErrorEventArgs
  fullName: System.Windows.Forms.DataGridViewDataErrorEventArgs
  type: Class
  summary: "Biedt gegevens voor de <xref href=&quot;System.Windows.Forms.DataGridView.DataError&quot;> </xref> gebeurtenis."
  remarks: "Afhandeling van de <xref:System.Windows.Forms.DataGridView.DataError>gebeurtenis kunt u de ingang-uitzonderingen die door de code buiten uw invloedssfeer (bijvoorbeeld door een externe gegevensbron).</xref:System.Windows.Forms.DataGridView.DataError> Gebruik de <xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A>eigenschap om te bepalen van de status van de <xref:System.Windows.Forms.DataGridView>op het moment van de uitzondering.</xref:System.Windows.Forms.DataGridView> </xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A> Gebruik de <xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A>eigenschap voor het ophalen van de uitzonderingsgegevens.</xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A> Als u wilt dat de uitzondering afgehandeld door aanvullende gebeurtenis-handlers, stelt u de <xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A>eigenschap `true`.</xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A>       De <xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A>en <xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.RowIndex%2A>Eigenschappen normaal geven aan de cel waarin de fout is opgetreden.</xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.RowIndex%2A> </xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A> Als de fout in een externe gegevensbron optreedt, kan de gegevensbron echter niet geven de kolom waarin de fout is opgetreden. In dit geval de <xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A>eigenschap meestal de kolom van de huidige cel geeft aan op het moment van de fout.</xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A>"
  example:
  - "The following code example demonstrates how to respond to information provided by the DataGridViewDataErrorEventArgs class. This example is part of a larger example available in the <xref:System.Windows.Forms.DataGridViewComboBoxColumn> class overview topic.  \n  \n [!code-cpp[System.Windows.Forms.DataGridViewColumn_AllColumns_Bound_Employees#50](~/add/codesnippet/cpp/t-system.windows.forms.d_56_1.cpp)]\n [!code-cs[System.Windows.Forms.DataGridViewColumn_AllColumns_Bound_Employees#50](~/add/codesnippet/csharp/t-system.windows.forms.d_56_1.cs)]\n [!code-vb[System.Windows.Forms.DataGridViewColumn_AllColumns_Bound_Employees#50](~/add/codesnippet/visualbasic/t-system.windows.forms.d_56_1.vb)]"
  syntax:
    content: 'public class DataGridViewDataErrorEventArgs : System.Windows.Forms.DataGridViewCellCancelEventArgs'
  inheritance:
  - System.ComponentModel.CancelEventArgs
  - System.Windows.Forms.DataGridViewCellCancelEventArgs
  implements: []
  inheritedMembers:
  - System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex
  - System.Windows.Forms.DataGridViewCellCancelEventArgs.RowIndex
  platform:
  - net462
- uid: System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)
  id: '#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)'
  parent: System.Windows.Forms.DataGridViewDataErrorEventArgs
  langs:
  - csharp
  name: DataGridViewDataErrorEventArgs(Exception,Int32,Int32,DataGridViewDataErrorContexts)
  nameWithType: DataGridViewDataErrorEventArgs.DataGridViewDataErrorEventArgs(Exception,Int32,Int32,DataGridViewDataErrorContexts)
  fullName: System.Windows.Forms.DataGridViewDataErrorEventArgs.DataGridViewDataErrorEventArgs(Exception,Int32,Int32,DataGridViewDataErrorContexts)
  type: Constructor
  assemblies:
  - System.Windows.Forms
  namespace: System.Windows.Forms
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.Windows.Forms.DataGridViewDataErrorEventArgs&quot;> </xref> klasse."
  syntax:
    content: public DataGridViewDataErrorEventArgs (Exception exception, int columnIndex, int rowIndex, System.Windows.Forms.DataGridViewDataErrorContexts context);
    parameters:
    - id: exception
      type: System.Exception
      description: "De uitzondering die is opgetreden."
    - id: columnIndex
      type: System.Int32
      description: "De kolomindex van de cel die deze gebeurtenis treedt op de <xref href=&quot;System.Windows.Forms.DataGridView.DataError&quot;> </xref>."
    - id: rowIndex
      type: System.Int32
      description: "De rij-index van de cel die deze gebeurtenis treedt op de <xref href=&quot;System.Windows.Forms.DataGridView.DataError&quot;> </xref>."
    - id: context
      type: System.Windows.Forms.DataGridViewDataErrorContexts
      description: "Een bitsgewijze combinatie van <xref href=&quot;System.Windows.Forms.DataGridViewDataErrorContexts&quot;> </xref> waarden van het gegevenstype van de context waarin de fout is opgetreden."
  overload: System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Windows.Forms.DataGridViewDataErrorEventArgs.Context
  id: Context
  parent: System.Windows.Forms.DataGridViewDataErrorEventArgs
  langs:
  - csharp
  name: Context
  nameWithType: DataGridViewDataErrorEventArgs.Context
  fullName: System.Windows.Forms.DataGridViewDataErrorEventArgs.Context
  type: Property
  assemblies:
  - System.Windows.Forms
  namespace: System.Windows.Forms
  summary: "Meer informatie opgehaald over de status van de <xref href=&quot;System.Windows.Forms.DataGridView&quot;> </xref> wanneer de fout is opgetreden."
  remarks: "Leden van de <xref:System.Windows.Forms.DataGridViewDataErrorContexts>opsomming kan worden gecombineerd met behulp van de bitsgewijze `OR` operator vertegenwoordigt de status van een gegevensgebonden <xref:System.Windows.Forms.DataGridView>wanneer er een fout is opgetreden.</xref:System.Windows.Forms.DataGridView> </xref:System.Windows.Forms.DataGridViewDataErrorContexts> Bijvoorbeeld, als een gebruiker krijgt een ongeldige celwaarde (zoals een naam typen in een cel die een datum vereist) en selecteert u vervolgens een andere cel de <xref:System.Windows.Forms.DataGridView>wordt geprobeerd de ongeldige celwaarde doorvoeren.</xref:System.Windows.Forms.DataGridView> Als het doorvoeren is mislukt, de <xref:System.Windows.Forms.DataGridView>verhoogt een <xref:System.Windows.Forms.DataGridView.DataError>gebeurtenis waarvan de eigenschap Context een waarde van <xref:System.Windows.Forms.DataGridViewDataErrorContexts>en <xref:System.Windows.Forms.DataGridViewDataErrorContexts>.</xref:System.Windows.Forms.DataGridViewDataErrorContexts> heeft</xref:System.Windows.Forms.DataGridViewDataErrorContexts> </xref:System.Windows.Forms.DataGridView.DataError> </xref:System.Windows.Forms.DataGridView>"
  example:
  - "The following code example demonstrates how to investigate the error context. This example is part of a larger example available in the <xref:System.Windows.Forms.DataGridViewComboBoxColumn> class overview topic.  \n  \n [!code-cpp[System.Windows.Forms.DataGridViewColumn_AllColumns_Bound_Employees#50](~/add/codesnippet/cpp/p-system.windows.forms.d_134_1.cpp)]\n [!code-cs[System.Windows.Forms.DataGridViewColumn_AllColumns_Bound_Employees#50](~/add/codesnippet/csharp/p-system.windows.forms.d_134_1.cs)]\n [!code-vb[System.Windows.Forms.DataGridViewColumn_AllColumns_Bound_Employees#50](~/add/codesnippet/visualbasic/p-system.windows.forms.d_134_1.vb)]"
  syntax:
    content: public System.Windows.Forms.DataGridViewDataErrorContexts Context { get; }
    return:
      type: System.Windows.Forms.DataGridViewDataErrorContexts
      description: "Een bitsgewijze combinatie van de <xref href=&quot;System.Windows.Forms.DataGridViewDataErrorContexts&quot;> </xref> waarden waarmee de context waarin de fout is opgetreden."
  overload: System.Windows.Forms.DataGridViewDataErrorEventArgs.Context*
  exceptions: []
  platform:
  - net462
- uid: System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception
  id: Exception
  parent: System.Windows.Forms.DataGridViewDataErrorEventArgs
  langs:
  - csharp
  name: Exception
  nameWithType: DataGridViewDataErrorEventArgs.Exception
  fullName: System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception
  type: Property
  assemblies:
  - System.Windows.Forms
  namespace: System.Windows.Forms
  summary: "Hiermee haalt u de uitzondering die staat voor de fout."
  remarks: "Voor het afhandelen van de uitzondering voor meer dan één <xref:System.Windows.Forms.DataGridViewDataErrorEventHandler>, genereert de uitzondering opnieuw aan het einde van een handler door in te stellen de <xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A>eigenschap `true`.</xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A> </xref:System.Windows.Forms.DataGridViewDataErrorEventHandler>"
  example:
  - "The following code example demonstrates checking the Exception property. This example is part of a larger example available in the <xref:System.Windows.Forms.DataGridViewComboBoxColumn> class overview topic.  \n  \n [!code-cpp[System.Windows.Forms.DataGridViewColumn_AllColumns_Bound_Employees#50](~/add/codesnippet/cpp/p-system.windows.forms.d_249_1.cpp)]\n [!code-cs[System.Windows.Forms.DataGridViewColumn_AllColumns_Bound_Employees#50](~/add/codesnippet/csharp/p-system.windows.forms.d_249_1.cs)]\n [!code-vb[System.Windows.Forms.DataGridViewColumn_AllColumns_Bound_Employees#50](~/add/codesnippet/visualbasic/p-system.windows.forms.d_249_1.vb)]"
  syntax:
    content: public Exception Exception { get; }
    return:
      type: System.Exception
      description: "Een <xref:System.Exception>die staat voor de fout.</xref:System.Exception>"
  overload: System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception*
  exceptions: []
  platform:
  - net462
- uid: System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException
  id: ThrowException
  parent: System.Windows.Forms.DataGridViewDataErrorEventArgs
  langs:
  - csharp
  name: ThrowException
  nameWithType: DataGridViewDataErrorEventArgs.ThrowException
  fullName: System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException
  type: Property
  assemblies:
  - System.Windows.Forms
  namespace: System.Windows.Forms
  summary: "Opgehaald of ingesteld waarmee wordt aangegeven of de uitzondering na de <xref href=&quot;System.Windows.Forms.DataGridViewDataErrorEventHandler&quot;> </xref> gemachtigde klaar is."
  remarks: "Deze eigenschap instellen op `false` als de <xref:System.Windows.Forms.DataGridViewDataErrorEventHandler>heeft afgehandeld met als uitzondering dat wordt vertegenwoordigd door de <xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A>eigenschap, en u niet wilt doorgeven van de uitzondering naar een andere <xref:System.Windows.Forms.DataGridViewDataErrorEventHandler>waarmee kan worden verwerkt de <xref:System.Windows.Forms.DataGridView.DataError>gebeurtenis.</xref:System.Windows.Forms.DataGridView.DataError> </xref:System.Windows.Forms.DataGridViewDataErrorEventHandler> </xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A> </xref:System.Windows.Forms.DataGridViewDataErrorEventHandler>       Dit kan nuttig zijn voor de uitzondering is doorgegeven en inspecteer de stacktracering voor informatie over de context van de fout zijn voor foutopsporing."
  example:
  - "The following code example demonstrates using the ThrowException property to indicate that the exception should not be thrown. This example is part of a larger example available in the <xref:System.Windows.Forms.DataGridViewComboBoxColumn> class overview topic.  \n  \n [!code-cpp[System.Windows.Forms.DataGridViewColumn_AllColumns_Bound_Employees#50](~/add/codesnippet/cpp/p-system.windows.forms.d_202_1.cpp)]\n [!code-cs[System.Windows.Forms.DataGridViewColumn_AllColumns_Bound_Employees#50](~/add/codesnippet/csharp/p-system.windows.forms.d_202_1.cs)]\n [!code-vb[System.Windows.Forms.DataGridViewColumn_AllColumns_Bound_Employees#50](~/add/codesnippet/visualbasic/p-system.windows.forms.d_202_1.vb)]"
  syntax:
    content: public bool ThrowException { get; set; }
    return:
      type: System.Boolean
      description: "<xref uid=&quot;langword_csharp_true&quot; name=&quot;true&quot; href=&quot;&quot;></xref>Als de uitzondering moet worden geretourneerd; anders <xref uid=&quot;langword_csharp_false&quot; name=&quot;false&quot; href=&quot;&quot;> </xref>. De standaardwaarde is <xref uid=&quot;langword_csharp_false&quot; name=&quot;false&quot; href=&quot;&quot;> </xref>."
  overload: System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException*
  exceptions:
  - type: System.ArgumentException
    commentId: T:System.ArgumentException
    description: "Wanneer deze eigenschap is ingesteld op <xref uid=&quot;langword_csharp_true&quot; name=&quot;true&quot; href=&quot;&quot;> </xref>, wordt de <xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception*>waarde voor eigenschap <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>.</xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception*>"
  platform:
  - net462
references:
- uid: System.Windows.Forms.DataGridViewCellCancelEventArgs
  isExternal: false
  name: System.Windows.Forms.DataGridViewCellCancelEventArgs
- uid: System.ArgumentException
  isExternal: true
  name: System.ArgumentException
- uid: System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)
  parent: System.Windows.Forms.DataGridViewDataErrorEventArgs
  isExternal: false
  name: DataGridViewDataErrorEventArgs(Exception,Int32,Int32,DataGridViewDataErrorContexts)
  nameWithType: DataGridViewDataErrorEventArgs.DataGridViewDataErrorEventArgs(Exception,Int32,Int32,DataGridViewDataErrorContexts)
  fullName: System.Windows.Forms.DataGridViewDataErrorEventArgs.DataGridViewDataErrorEventArgs(Exception,Int32,Int32,DataGridViewDataErrorContexts)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Int32
  parent: System
  isExternal: true
  name: Int32
  nameWithType: Int32
  fullName: System.Int32
- uid: System.Windows.Forms.DataGridViewDataErrorContexts
  parent: System.Windows.Forms
  isExternal: false
  name: DataGridViewDataErrorContexts
  nameWithType: DataGridViewDataErrorContexts
  fullName: System.Windows.Forms.DataGridViewDataErrorContexts
- uid: System.Windows.Forms.DataGridViewDataErrorEventArgs.Context
  parent: System.Windows.Forms.DataGridViewDataErrorEventArgs
  isExternal: false
  name: Context
  nameWithType: DataGridViewDataErrorEventArgs.Context
  fullName: System.Windows.Forms.DataGridViewDataErrorEventArgs.Context
- uid: System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception
  parent: System.Windows.Forms.DataGridViewDataErrorEventArgs
  isExternal: false
  name: Exception
  nameWithType: DataGridViewDataErrorEventArgs.Exception
  fullName: System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception
- uid: System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException
  parent: System.Windows.Forms.DataGridViewDataErrorEventArgs
  isExternal: false
  name: ThrowException
  nameWithType: DataGridViewDataErrorEventArgs.ThrowException
  fullName: System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException
- uid: System.Boolean
  parent: System
  isExternal: true
  name: Boolean
  nameWithType: Boolean
  fullName: System.Boolean
- uid: System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor*
  parent: System.Windows.Forms.DataGridViewDataErrorEventArgs
  isExternal: false
  name: DataGridViewDataErrorEventArgs
  nameWithType: DataGridViewDataErrorEventArgs.DataGridViewDataErrorEventArgs
- uid: System.Windows.Forms.DataGridViewDataErrorEventArgs.Context*
  parent: System.Windows.Forms.DataGridViewDataErrorEventArgs
  isExternal: false
  name: Context
  nameWithType: DataGridViewDataErrorEventArgs.Context
- uid: System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception*
  parent: System.Windows.Forms.DataGridViewDataErrorEventArgs
  isExternal: false
  name: Exception
  nameWithType: DataGridViewDataErrorEventArgs.Exception
- uid: System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException*
  parent: System.Windows.Forms.DataGridViewDataErrorEventArgs
  isExternal: false
  name: ThrowException
  nameWithType: DataGridViewDataErrorEventArgs.ThrowException
